README.md

```diff
@@ -1 +1,165 @@
-# Portfolio-
\ No newline at end of file
+<<<<<<< HEAD
+
+# Portfolio: setup and mobile version skeleton
+
+For this repository, I've created html templates and styling files in order to built a responsive portfolio , which can help to easy explorate profiles from differents devices.
+
+![Live Demo Link](<https://github.com/naji-97/Portfolio-/assest/img/Screenshot> 2022-12-02 111444.png/)
+
+## Built With 🔨
```

Comment:
- [ ] OPTIONAL] Kindly make sure your `README` template follows the new Microverse [template](https://github.com/microverseinc/readme-template/blob/master/README.md) 🙏.